blogpost_14.html

```diff
@@ -201,25 +189,20 @@ <h2><a href="https://docs.xfce.org/xfce/xfce4-panel/start" target="_blank">xfce4
             <div class="t1">
 
                 <h3>General</h3>
-                  <p>The auto-hide feature was improved in multiple ways and now works much more reliable.</p>
-                  <p class="note">@Gael: You maybe want to add details on auto-hide improvements?</p>
-                  <p>The minimum button size was reduced, in order to improve the layout for very small panels.</p>
                   <p>The border width of the panel is now configurable.</p>
-                  <p>It is now possible to set a background color or background image for the panel.</p>
-                  <img class="full-img" src="images/blogpost_14/panel_color.png" alt="xfce4-panel - colored background"/>
```

Comment:
Uh, looks like I somewhat overlooked that option in the xfce4.18 panel and as such thought it would be new  :cat: